share/man/man1/intro.1

```diff
@@ -46,32 +49,57 @@ compilers and compiler tools,
 formatted output tools,
 and line printer commands.
 .Pp
+Tens of thousands of additional commands are available to be installed
+with
+.Xr pkg 8 ,
+or compiled with the
+.Xr ports 7
+collection.
+Some of which include web browsers, office suites, calendars,
+conferencing utilities, integrated development environments,
+media players, audio and video processing suites, etc.
+.Pp
 All commands set a status value upon exit which may be tested
 to see if the command completed normally.
 Traditionally, the value 0 signifies successful
 completion of the command, while a value >0 indicates an error.
 Some commands attempt to describe the nature of the failure by using
 exit codes as defined in
 .Xr sysexits 3 ,
-while others simply set the status to an arbitrary value >0 (typically 1).
+while others simply set the status to an arbitrary value >0
+.Pq typically 1 .
+.Sh FILES
+.Bl -tag -width "/usr/local/bin/tab" -compact
+.It Pa /bin/
+Commands fundamental to single- and multi-user modes.
```

Comment:
It should be
```diff
- single- and multi-user modes
+ single and multi-user modes
```
(an extra hyphen was added)

Reply:
Actually, what's there is correct, and the advice is in error.
This is a weird edge-case of English. When you have two words that would be hyphenated, like single-user and multi-user, when you combine them with and, you append a hyphen to the end of the first one, as was sone here. I double checked this with my wife, who edits for a living, has been an English professor and has a life long obesssion with pedantic American English grammar rules.

Reply:
For more info: https://english.stackexchange.com/questions/113422/how-to-use-hyphens-appropriately-when-listing-multiple-hyphenated-terms

Some style guides may omit them, but there are cases where doing so can lead to ambiguity and they indeed should be kept. If folks really don't like the "dangling" hyphen for some reason just repeat modes: _single-user and multi-user modes_

Reply:
I used the hyphen on purpose, but was willing to remove it because I didn't think it would introduce any ambiguity.

We use the dangling hyphen in uefi(8) ("provides boot- and run-time services") and I think this is the most elegant and terse way to do it.

Repeating user twice folds the description onto 2 lines on standard console. I've been taking great care to avoid that where possible.

Reply:
This is not exactly an "error". I don't believe I've seen usage of "single- and multi-user" in books (most of the time if there are only two words, have hyphens, they just write it as: single-user and multi-user or if there are more words, they just write it as: first, second, third, and forth-fold).

Both usage are okay, we avoid dangling hyphen as it possibly feels redundant. But again, I'm not reading pure American/British English literature as it's not my first language..